python/tk_desktop/project_model.py

```diff
@@ -214,6 +215,22 @@ class SgProjectModel(ShotgunModel):
     thumbnail_updated = QtCore.Signal(QtGui.QStandardItem)
     project_launched = QtCore.Signal()
 
+    _supports_project_templates = None
+
+    @staticmethod
```

Comment:
Sorry, old habits die hard. I only started using @classmethod recently and therefore often forget about it. The code looks a lot better using @classmethod now. Thanks.